cpp/roaring.hh

```diff
@@ -42,6 +42,23 @@ namespace roaring {
 
 class RoaringSetBitForwardIterator;
 
+/**
+ * A bit of context usable with `*Bulk()` functions.
```

Comment:
Why not write "with the AddBulk function" ?

Reply:
Emm here I port it from the text below:

> A bit of context usable with `roaring_bitmap_*_bulk()` functions

And I found that there are two bulk functions: `roaring_bitmap_add_bulk` and `roaring_bitmap_contains_bulk`. Currently I only implement `roaring_bitmap_add_bulk`. But I think when I implement `roaring_bitmap_contains_bulk` wrapper as `containsBulk` in the future, `*Bulk()` is also suitable

Reply:
If it is only a few lines of code, I would add `roaring_bitmap_contains_bulk` and do just one PR, and we would be done with this issue.

Reply:
Sure, let me add it

Reply:
done